core/domain/state_domain.py

```diff
@@ -714,6 +714,190 @@ def validate(self):
                 self.needs_update)
 
 
+class TranslationScript(object):
```

Comment:
Let's call this WrittenTranslation instead. "Script" sounds like a str.

Reply:
Done!